recipes/omniscidb-cuda/recipe/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "omniscidb-cuda" %}
+{% set version = "4.8.0" %}
+{% set sha256 = "06ec0990d362963fc5e02ad64bc90d99ad0d226014354de4566af267d33443b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/omnisci/omniscidb/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    #- {{ compiler('cxx') }}
+    #- {{ compiler('c') }}
```

Comment:
I think you need to have the compilers listed here to get the proper deps on install

Reply:
done

Reply:
done